src/overlays/actors/ovl_En_Ru/z_en_ru.h

```diff
@@ -10,6 +10,8 @@ struct EnRu;
 
 typedef void (*EnRuActionFunc)(struct EnRu*, PlayState*);
 
+#define RU_BODYPARTSPOS_SIZE 15
```

Comment:
`COUNT` would be better since the size is `0xC` times larger.

Reply:
Ah, true